tests/test_e2e_60_of_multi_table.py

```diff
@@ -0,0 +1,430 @@
+        response = requests.post(api_url)
+        assert response.status_code == 200
+        time.sleep(10)
+
+        # Get pipeline dictionary
+        api_url = f"{KYTOS_API}{OF_MULTI_TABLE_API}/{data['id']}"
+        response = requests.get(api_url)
+        pipeline_data = response.json()
+        assert response.status_code == 200
+        assert pipeline_data["status"] == "enabled"
+
+        # Assert installed flows
+        s1 = self.net.net.get('s1')
+        flows_s1 = s1.dpctl('dump-flows').splitlines()
+        assert "table=0" in flows_s1[0]
+        assert 'priority=0 actions=resubmit(,1)' in flows_s1[0]
+        assert "table=1" in flows_s1[1]
+        assert 'actions=CONTROLLER:65535' in flows_s1[1]
+        assert "table=1" in flows_s1[2]
+        assert 'actions=CONTROLLER:65535' in flows_s1[2]
+        assert "table=1" in flows_s1[3]
+        assert 'priority=0 actions=resubmit(,2)' in flows_s1[3]
+        assert "table=2" in flows_s1[4]
+        assert 'dl_type=0x88cc actions=CONTROLLER:65535' in flows_s1[4]
+        assert "table=2" in flows_s1[5]
+        assert 'priority=0 actions=resubmit(,3)' in flows_s1[5]
+        assert "table=3" in flows_s1[6]
+        assert 'dl_vlan=100 actions=output:"s1-eth2"' in flows_s1[6]
+        assert "table=3" in flows_s1[7]
+        assert 'priority=0 actions=resubmit(,4)' in flows_s1[7]
+        assert "table=4" in flows_s1[8]
+        assert 'actions=mod_vlan_vid:100,output:"s1-eth1"' in flows_s1[8]
+
+        self.net.start_controller(clean_config=False)
+        self.net.wait_switches_connect()
+        time.sleep(10)
+
+        # Assert installed flows
+        s1 = self.net.net.get('s1')
+        flows_s1 = s1.dpctl('dump-flows').splitlines()
+        assert "table=0" in flows_s1[0]
+        assert 'priority=0 actions=resubmit(,1)' in flows_s1[0]
+        assert "table=1" in flows_s1[1]
+        assert 'actions=CONTROLLER:65535' in flows_s1[1]
+        assert "table=1" in flows_s1[2]
+        assert 'actions=CONTROLLER:65535' in flows_s1[2]
+        assert "table=1" in flows_s1[3]
+        assert 'priority=0 actions=resubmit(,2)' in flows_s1[3]
+        assert "table=2" in flows_s1[4]
+        assert 'dl_type=0x88cc actions=CONTROLLER:65535' in flows_s1[4]
+        assert "table=2" in flows_s1[5]
+        assert 'priority=0 actions=resubmit(,3)' in flows_s1[5]
+        assert "table=3" in flows_s1[6]
+        assert 'dl_vlan=100 actions=output:"s1-eth2"' in flows_s1[6]
+        assert "table=3" in flows_s1[7]
+        assert 'priority=0 actions=resubmit(,4)' in flows_s1[7]
+        assert "table=4" in flows_s1[8]
+        assert 'actions=mod_vlan_vid:100,output:"s1-eth1"' in flows_s1[8]
+
+        # Return to default pipeline
+        # Disabled pipeline
+        api_url = f"{KYTOS_API}{OF_MULTI_TABLE_API}/{data['id']}/disable"
+        response = requests.post(api_url)
+        assert response.status_code == 200
+        time.sleep(10)
```

Comment:
After this point here, could you also run `dump-flows` agains and assert all expected flows and the number of flows too? I'd like to make sure that the observed issue that I mentioned [on this comment during a code review](https://github.com/kytos-ng/of_multi_table/pull/13#issuecomment-1569268215) doesn't happen here.